README.md

```diff
@@ -1,130 +1,147 @@
+   > - Please check [nf-core/configs](https://github.com/nf-core/configs#documentation) to see if a custom config file to run nf-core pipelines already exists for your Institute. If so, you can simply use `-profile <institute>` in your command. This will enable either `docker` or `singularity` and set the appropriate execution settings for your local compute environment.
+   > - If you are using `singularity`, please use the [`nf-core download`](https://nf-co.re/tools/#downloading-pipelines-for-offline-use) command to download images first, before running the pipeline. Setting the [`NXF_SINGULARITY_CACHEDIR` or `singularity.cacheDir`](https://www.nextflow.io/docs/latest/singularity.html?#singularity-docker-hub) Nextflow options enables you to store and re-use the images from a central location for future pipeline runs.
+   > - If you are using `conda`, it is highly recommended to use the [`NXF_CONDA_CACHEDIR` or `conda.cacheDir`](https://www.nextflow.io/docs/latest/conda.html) settings to store the environments in a central location for future pipeline runs.
 
-```bash
-nextflow run nf-core/rnafusion -profile <docker/singularity/institute> \
-  --reads '*_R{1,2}.fastq.gz' \
-  --genomes_base 'reference_path_from_above'
-  --arriba --star_fusion --fusioncatcher --ericscript --pizzly --squid \
-  --arriba_vis --fusion_inspector
+4. Start running your own analysis!
+
+```console
+nextflow run nf-core/rnafusion --input samplesheet.csv --outdir <OUTDIR> --genome GRCh38 --all -profile <docker/singularity/podman/shifter/charliecloud/institute>
 ```
 
-See [usage docs](docs/usage.md) for all of the available options when running the pipeline.
+> Note that paths need to be absolute and that runs with conda are not supported.
 
 ## Documentation
 
-The nf-core/rnafusion pipeline comes with documentation about the pipeline, found in the `docs/` directory:
+The nf-core/rnafusion pipeline comes with documentation about the pipeline [usage](https://nf-co.re/rnafusion/usage), [parameters](https://nf-co.re/rnafusion/parameters) and [output](https://nf-co.re/rnafusion/output).
 
-1. [Installation](https://nf-co.re/usage/installation)
-2. Pipeline configuration
-    * [Download references](docs/references.md)
-    * [Local installation](https://nf-co.re/usage/local_installation)
-    * [Adding your own system config](https://nf-co.re/usage/adding_own_config)
-3. [Running the pipeline](docs/usage.md)
-4. [Output and how to interpret the results](docs/output.md)
-5. [Troubleshooting](https://nf-co.re/usage/troubleshooting)
+## Credits
 
-Use predefined configuration for desired Institution cluster provided at [nfcore/config](https://github.com/nf-core/configs) repository.
+nf-core/rnafusion was originally written by Martin Proks ([@matq007](https://github.com/matq007)), Praveen Raj ([@praveenraj2018](https://github.com/praveenraj2018))
```

Comment:
Was Praveen really doing the effort here? I'm not judging, but the code base tells me very different things.

Reply:
Should have Martin and then Maxime + you Annick here as authors tbh. Others maybe just as a in "acknowledgements" for contributing to testing / running the pipeline / providing feedback (Phil, Me, Rickard, just looking at certain bits / discussion on Slack).